app/views/layouts/application.html.haml

```diff
@@ -35,6 +35,7 @@
 
     = stylesheet_link_tag '/inert.css', skip_pipeline: true, media: 'all', id: 'inert-style'
     = stylesheet_link_tag custom_css_path, skip_pipeline: true, host: root_url, media: 'all'
+    = stylesheet_link_tag '/custom.css', skip_pipeline: true, media: 'all', id: 'custom-style'
```

Comment:
Just to confirm, this means we are importing the custom css file and we no longer need to add an import statement over at:

 > _Appearance_ > _Custom CSS_ input? `(@import url("./custom.css");`
 
 right?

Reply:
Correct.